### PR TITLE
Add instruction to build the app

### PR DESCRIPTION
**Pull request by cjsaez**

### CODE REVIEW

**Review by rlaguilar — CHANGES_REQUESTED** [review]
The instructions don't say where I can get the API key from. Have a look at the different alternatives that can be used to share secret values among team members.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Pelis
 
-App using the TMDB endpoints.
\ No newline at end of file
+App using the TMDB endpoints.
+
+In order to android studio will can build the app, add to local.properties file the api key with "API_KEY="{value}"" format.
```
Comment (rlaguilar) [inline README.md:5]: Are you sure `"API_KEY="{value}""` is the right value to add?

**Review by rlaguilar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cjsaez) [reply README.md]: I don't understand the question.

Comment (rlaguilar) [reply README.md]: If the api key is `abcde`, what do we need to add to the `local.properties` file?

**Comment by cjsaez** [comment]
Basically there are three ways, environment variables, local properties or getting from a third service. The first two options are similar in terms of security but the third one adds a necessary call to conform the rest of the requests so it adds complexity.

File: README.md (hunk 1 above)
Comment (cjsaez) [reply README.md]: API_KEY="abcde"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
```
Comment (rlaguilar) [inline README.md:5]:
```suggestion
In order to android studio will can build the app, add to local.properties file the api key with `API_KEY="{value}"` format. 
```

**Review by rlaguilar — APPROVED** [review]

**Pull request merged**